src/components/App.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
```

Comment:
**[OPTIONAL]**
Disabling the eslint rule would make you miss out on the object of using the specific rule that has been disabled.